generator/.DevConfigs/48cf8831-8233-4a7e-853c-5352f8ca948d.json

```diff
@@ -0,0 +1,18 @@
+{
+  "core": {
+    "updateMinimum": true,
```

Comment:
Do you need the Core section here if the only service being changed is S3? Also, would it make sense to make this a `Minor` change for S3 since it's a new feature?

Reply:
i can update to minor